Scala2-Functional-Programming/Week6/forcomp/src/main/scala/forcomp/Anagrams.scala

```diff
@@ -92,7 +109,16 @@ object Anagrams extends AnagramsInterface {
    *  Note: the resulting value is an occurrence - meaning it is sorted
    *  and has no zero-entries.
    */
-  def subtract(x: Occurrences, y: Occurrences): Occurrences = ???
+  def subtract(x: Occurrences, y: Occurrences): Occurrences = {
+    val yMap = y.toMap.withDefaultValue(0)
+    for {
+      (xc,xi) <- x
```

Comment:
Lost you here with those variable names...
Please change

Reply:
Done